app/src/main/res/drawable/ic_popup.xml

```diff
@@ -0,0 +1,11 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:autoMirrored="true"
+    android:tint="#000000"
```

Comment:
```suggestion
    android:tint="?attr/colorControlNormal"
```

Reply:
It looked pretty weird on light theme, I'll have a try...

Reply:
Gotcha!
![Screenshot_20221124-132913.png](https://user-images.githubusercontent.com/60344409/203784975-6a7eef6b-74a5-45a6-b6b4-8ba976d49703.png)
![Screenshot_20221124-132922.png](https://user-images.githubusercontent.com/60344409/203785001-3cf31ad8-f59f-4b94-a39b-ef19086dd855.png)

Reply:
Looks good